NEW_COMPONENT.md

```diff
@@ -4,33 +4,39 @@ In this guide, we'll explore the library and documentation code base and create
 
 ### Code Base Structure
 
-This repository contains two separate projects - the component library and the documentation application.  Different build tasks are used, depending on which project we're building.  However, the documentation app consumes the library source directly, so there's no need to compile the library to test changes you're making to a component - simply running the documentation app locally will serve changes to the library immediately.
+This repository contains two separate projects - the component library and the documentation application. Different build tasks are used, depending on which project we're building. However, the documentation app consumes the library source directly, so there's no need to compile the library to test changes you're making to a component - simply running the documentation app locally will serve changes to the library immediately.
 
-The documentation code base lies in the `docs` directory and the library source is in the `library` directory.
+The documentation code base lies in the `docs` directory and the library source is in the `libs` directory.
 
 ## Create a new module in the Library
 
-First, let's generate an empty module in the library.  Each library component gets its own module, so end users can import only the modules for specific components they intend to use, if they don't want to import the entire library.
+First, let's generate an empty module in the library. Each library component gets its own module, so end users can import only the modules for specific components they intend to use, if they don't want to import the entire library.
 
-The library's source lies in the `library` directory.  From the root of the repo, cd into the library source:
+## Step1: Cd libs
 
-`cd library`
+The library's source lies in the `libs` directory. From the root of the repo, cd into the library source:
 
```

Comment:
More explanation needs to be provided for this step - what does cd into the library source mean?

Reply:
cd is a terminal command it means go into the directory